ui/src/components/search-result.tsx

```diff
@@ -1,149 +1,174 @@
+/* 
+This file contains the code for the <SearchResult/> Component.
+See "SearchResultProps" on this document for the the properties this component
+uses.
+*/
 
 import React from 'react';
 
+// If not working, run "npm install react-icons"
 import { FaConfluence, FaSlack, FaGoogleDrive } from "react-icons/fa";
 
-import BlueFolder from '../assets/images/blue-folder.svg';
-import Slack from '../assets/images/slack.svg';
-import Confluence from '../assets/images/confluence.svg';
-import GoogleDrive from '../assets/images/google-drive.svg';
+// This imports the full-color icons
+import BlueFolder from './assets/images/blue-folder.svg';
+import Slack from './assets/images/slack.svg';
+import Confluence from './assets/images/confluence.svg';
+import Drive from './assets/images/google-drive.svg'
```

Comment:
Rename back to GoogleDrive, we might add different kinds of Drives in the future.